README.md

```diff
@@ -907,42 +907,25 @@ This allows multiple pipelines to use the same code for share tools and gives a
 
 The nf-core DSL2 modules repository is at <https://github.com/nf-core/modules>
 
+### Running the module commands
+
+The `nf-core/tools` package contains a suite of commands for working DSL2 modules in your pipeline. The modules commands use `git` to interface
+with the `nf-core/modules` repository. Therefore you need to have have `git` installed to use the modules commands. You can then use the
+the modules commands for a variety of tasks such as creating new DSL2 modules, viewing and getting metadata about the modules in your pipeline,
+and installing and updating modules from a remote git repository.
+
```

Comment:
I think we can remove this. Everyone should have git already, I hope.. And the error message should be self-explanatory if not.